config/routes.rb

```diff
@@ -15,6 +15,7 @@
        namespace :traveler do 
               resources :journeys
        end 
+       get 'journey_list', action: :journey_list, controller: 'traveler/journeys'
```

Comment:
follow proper way to make route @aleemchishti

Reply:
didn't work otherwise. followed railsguide syntax for namespace

Reply:
get 'journey_list',to: 'traveler/journeys#journey_list' 
worked with this